ring_doorbell/doorbot.py

```diff
@@ -289,16 +289,25 @@ def live_streaming_json(self):
                 pass
         return None
 
-    def recording_download(self, recording_id, filename=None, override=False):
+    def recording_download(self, recording_id, filename=None,
+                           override=False, timeout=None):
         """Save a recording in MP4 format to a file or return raw."""
+
+        # Configure timeout if specified
+        if timeout is None:
+            download_timeout = DEFAULT_VIDEO_DOWNLOAD_TIMEOUT
+        else:
+            download_timeout = timeout
+
```

Comment:
You can remove this block because the timeout has already been initialized by the DEFAULT_VIDEO_DOWNLOAD_TIMEOUT. If the user decides to use a different value, it will have it already received by parameter.